test/test.js

```diff
@@ -1720,4 +1720,46 @@ describe('sanitizeHtml', function() {
 
     assert.equal(sanitizedHtml, expectedOutput);
   });
+  it('should call onOpenTag and onCloseTag callbacks', () => {
```

Comment:
I'm wondering if we should modify this test to include a tag with an implied close?

Reply:
Included an implicitly closed tag